specification/e-referrals-service-api.yaml

```diff
@@ -1148,7 +1148,70 @@ paths:
         '502':
           $ref: 'components/schemas/responses/bookOrDeferAppointment/502Response.yaml'      
         '504':
-          $ref: 'components/schemas/responses/bookOrDeferAppointment/504Response.yaml'             
+          $ref: 'components/schemas/responses/bookOrDeferAppointment/504Response.yaml'
+  /STU3/v1/ReferralRequest/{ubrn}/$ers.generateCRI:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to generate a PDF file that summarises clinical information for a referral.
+
+        ## Pre-requisites
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
+          - `SERVICE_PROVIDER_CLINICIAN`
```

Comment:
Do we want to publicize that SPCs can access this endpoint via the API-M platform given that we haven't fleshed out the provider APIs yet? I'm not sure what we've done for the other endpoints around this as the endpoint will actually work currently for users using the SPC business function

Reply:
We've tried to put the correct roles on all endpoints (including spc) simply so that we don't have to revisit them all when creating spec for provider side.